test/extended/deployments/util.go

```diff
@@ -377,7 +388,11 @@ func waitForSyncedConfig(oc *exutil.CLI, name string, timeout time.Duration) err
 		if err != nil {
 			return false, err
 		}
-		return appsinternalutil.HasSynced(config, generation), nil
+		externalConfig := &appsapiv1.DeploymentConfig{}
```

Comment:
why don't we switch the client to external one instead?

Reply:
i think i switched this to external in next commit